.github/workflows/Documentation.yml

```
@@ -0,0 +1,22 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: '1.1'
```

Should this be? Probably doesnt matter as tests should pass on all versions

```suggestion
          version: '1'
```

---

I think printed output can sometimes change between Julia minor versions, so the doctests might suddenly start failing if we don't fix this to a specific version. At this point in time, we should probably fix it at 1.6, though...

---

...which indeed happens to break the doctests as tested locally due to changed printing of `Array{Int64,1}` as `Vector{Int64}`. So I'd propose leaving this at 1.1 for now and then doing another PR for bumping it to 1.6.

---

Excellent, thanks for taking the time to explain in detail.